docs/pages/identity-security/usage/alerts.mdx

```diff
@@ -0,0 +1,139 @@
+
+Set up GitHub detections: [GitHub Settings](../integrations/github.mdx)
+
+### Okta detections
+
+Track identity and access management events in your Okta environment:
+
+#### High severity
+- Admin MFA disabled
+- All MFA factors reset
+- OAuth token reuse patterns
+- Rate limit violations
+- Sign-on policy evaluations
+- IDP lifecycle changes
+- API token creation/revocation
+- Security threat events
+
+#### Medium severity
+- Excessive MFA failures
+- Multiple password reset attempts
+- Dormant account access
+- Support-initiated password or MFA resets
+
+### Teleport detections
+
+Monitor Teleport-specific security events:
+
+#### High severity
+- Root SSH session initiation
+- Authentication without MFA for local accounts
+- Unusual authentication failure patterns
+- Role creation/updates/deletions
+- Connector updates
+- Unusual session commands
+
+Set up Teleport detections: [Teleport Settings](../integrations/teleport.mdx)
+
+### Cross-platform detections
+
+#### Impossible travel detection (GitHub, Okta, Teleport)
+- Severity: High
+- Triggers when login attempts occur from multiple geographic locations within a timeframe that makes legitimate travel physically impossible
+
+### Current Capabilities
+- View alerts across all integrated systems - Access a unified dashboard in the Teleport Identity Security  that displays all security alerts from AWS, GitHub, Okta, and Teleport in a single view, eliminating the need to check multiple platforms separately.
+- Filter by severity level - Narrow down the alerts list to show only Critical, High, Medium, or Low severity events, helping you prioritize which security issues to investigate first.
+- Investigate related events - View associated events, graph and context in the Investigate view, allowing you to understand the full scope of the security incident and trace related activities across your infrastructure.
+
+## Frequently asked questions
+
+### Are these detections available in Teleport Cloud?
+No, Identity Activity Center detections are only available in self-hosted Teleport Enterprise deployments. They will be coming to Teleport Enterprise Cloud in Q4 2025.
+
+### Can I write my own detections?
+No, detections are currently pre-configured per integration and cannot be customized.
+
+### Can I resolve an alert?
+Not currently, but we plan to add workflows to resolve, acknowledge, and mute alert types in future updates.
+
+### What determines the severity level?
+Severity levels are defined by the Identity Security Team based on the potential security impact. Feedback on severity assignments is welcome.
+
+### Can I forward these events to another service?
+This feature is planned for future release.
+
+### How can I request new detection types?
+For new detection requests based on customer needs, reach out to the Identity Security Team. Customer feedback is welcomed and helps prioritize new detection development.
```

Comment:
How do we expect the reader to know how to get in touch with the Identity Security team?